### ISSUE TITLE
WinUI Accessibility->CommandBarFlyout card: See more menu list is expanding automatically when image is invoked.

### ISSUE THREAD

**Issue by gregwoo-microsoft** [microsoft/WinUI-Gallery#876]
**Repro-Steps:**
1. Launch the WinUI app
2. Navigate to search box using tab key.
3. Type 'CommandBarFlyout' in search box and hit enter.
4. Navigate to CommandBarFlyout card and activate it.
5. Now navigate to image and hit enter.

**Actual Result:**
After invoking on image resize and move menu items are visible without user interaction.

**Expected Result:**
Resize and move menu list items should be visible after invoking see more button only.

**User impact:**
Without invoking the button if the menu list are be interacted without user interaction then keyboard user will be confused.

**Note**
When mouse pointer is placed out of the card, issue is repro'ing,
When mouse pointer is placed at the image in the card, issue is not repro'ing.

**Comment by evelynwu-msft** [comment]
This is by-design behavior in the app; the sample is written such that right-clicking on the image opens the flyout in [standard mode](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.primitives.flyoutshowmode?view=windows-app-sdk-1.2) so it is expanded.
